tools/fix_quota_id/fix_quota_id.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"os"
+
+	"github.com/alibaba/pouch/storage/quota"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dir       string
```

Comment:
please more description for all these variables clearly. That could really helps a lot.